File: Assignmentno20/task_2.java
import java.util.*;
class Logic
{
    void PrintReverse(int iValue)
    {
        int iCnt = 0;
        int iDigit = 0;
        iDigit = iValue%10;
        for(iCnt = iValue;iCnt>=1;iCnt--)
        {
            System.out.println(iCnt);
        }
        iValue = iValue/10;
    }

}
class task_2
{
    public static void main(String A[])
    {
        int iNo = 0;
        Scanner sobj = new Scanner(System.in);
        Logic lobj = new Logic();

        System.out.println("Enter the number:");
        iNo = sobj.nextInt();
        lobj.PrintReverse(iNo);



    }
}

File: Assignmentno71/task_2.c
#include<stdio.h>

int LargestDigit(int iValue)
{
     int iMax = 0;
     int iDigit = 0;

    if(iValue == 0)
    {
        return iMax;
    }
    iDigit = iValue %10;
    iMax = LargestDigit(iValue/10);
    if(iDigit > iMax)
    {
        return iDigit;
    }
    else
    {
        return iMax;
    }

}
int main()
{
    int iNo = 0 ;
    int iRet = 0;
    printf("Enter the number:\n");
    scanf("%d",&iNo);

    iRet = LargestDigit(iNo);
    printf("The largest digit is :%d\n",iRet);

    return 0;
}